cron.go

```diff
@@ -36,6 +36,9 @@ type Schedule interface {
 	// Next returns the next activation time, later than the given time.
 	// Next is invoked initially, and then each time the job is run.
 	Next(time.Time) time.Time
+	// Previous returns the next activation time, smaller or equal with the given time.
+	// Previous is invoked initially, and then each time the job is run.
+	Previous(time.Time) time.Time
```

Comment:
per https://github.com/robfig/cron/blob/e843a09e5b2db454d77aad25b1660173445fb2fc/cron.go#L53-L59

maybe it is better to rename this from `Previous` to `Prev`

Reply:
done